src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/IdentifiersShouldNotContainUnderscores.cs

```diff
@@ -167,6 +167,12 @@ public override void Initialize(AnalysisContext analysisContext)
                                     return;
                                 }
 
+                                if (methodSymbol.MethodKind == MethodKind.Conversion)
```

Comment:
Combine with the previous if

Reply:
Thank you! I'll keep it separate for readability. These are two different scenarios.

---
In reply to: [148387144](https://github.com/dotnet/roslyn-analyzers/pull/1329#discussion_r148387144) [](ancestors = 148387144)